### PR TITLE
derp: add server version to /debug, expvars

### PR DESCRIPTION
**Pull request by josharian**
This change is [Reviewable](https://reviewable.io/reviews/tailscale/tailscale/652)

### CODE REVIEW

File: tsweb/tsweb.go [hunk 1]
```diff
@@ -382,6 +383,9 @@ func varzHandler(w http.ResponseWriter, r *http.Request) {
 				dump(name+"_", kv)
 			})
 			return
+		case *expvar.String:
+			fmt.Fprintf(w, "# TYPE %s literal\n%s %v\n", name, name, v.Value())
```
Comment (bradfitz) [inline tsweb/tsweb.go:387]: Where is this documented?

Not at https://github.com/prometheus/docs/blob/master/content/docs/instrumenting/exposition_formats.md (as referenced on line 341 above)

What's the quoting rules for the string?

File: derp/derp_server.go [hunk 2]
```diff
@@ -103,6 +104,12 @@ type Server struct {
 	sentTo map[key.Public]map[key.Public]int64 // src => dst => dst's latest sclient.connNum
 }
 
+var expvarVersion expvar.String
```
Comment (bradfitz) [inline derp/derp_server.go:107]: just put this and the init down in the ExpVar method?

File: tsweb/tsweb.go (hunk 1 above)
Comment (josharian) [reply tsweb/tsweb.go:387]: Oops. I was looking at the querying docs. :( Thanks.

File: derp/derp_server.go (hunk 2 above)
Comment (josharian) [reply derp/derp_server.go]: Done.

**Pull request merged**